x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { AnalyticsTable } from './analytics_table';
+export { RecentQueriesTable } from './recent_queries_table';
+export { QueryClicksTable } from './query_clicks_table';
```

Comment:
FYI: I'm not super in love with `constants.tsx` as a way of sharing certain columns & UI properties between analytics table, but it's also difficult to see any other way to do so since the last 2 tables have such drastically different data structures. We'd have to go into our server-side API to tweak field names and so forth to do so, which I don't think is realistic for the migration right now.

Reply:
Yeah I don't mind it here. Or even a dedicated `columns.tsx` file or something.

Reply:
Ahh awesome suggestion, I definitely like `columns.tsx` over `constants.tsx` (or maybe even `shared_columns` to be super explicit 🤔). Thanks Jason, will make that change!

Reply:
7e3044e